SmallTasksTraining/task_2.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+
+int main() {
+    int sum_a = 0;
+    int b = 0;
+    int a;
+
+    for (int i = 1; i <= 10; ++i) {
```

Comment:
Could int sum_a = 0; be init here?

```suggestion
    for (int i = 1, int sum_a = 0;; i <= 10; ++i) {
```